test/karma.conf.js

```diff
@@ -0,0 +1,32 @@
+// This file only hook up on require calls to transpile the TypeScript.
+// If you're looking at this file to see Karma configuration, you should look at
+// karma.config.ts instead.
```

Comment:
In my defense, the browser-providers files is getting smaller and easier to read. The configuration itself doesn't fully benefit but might in the future if we add stuff.

I agree though that this is a small section of the repo that won't get touched regularly.